tests/action/identity.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2025 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package action
+
+import (
+	"context"
+	"time"
+
+	"github.com/sourcenetwork/immutable"
+	"github.com/stretchr/testify/require"
+
+	acpIdentity "github.com/sourcenetwork/defradb/acp/identity"
+	"github.com/sourcenetwork/defradb/tests/state"
+)
+
+const (
+	// AuthTokenExpiration is the expiration time for auth tokens.
+	AuthTokenExpiration = time.Minute * 1
+)
```

Comment:
question: Are people happy this being here or should this be in state?

Reply:
yes :) Thanks